src/main/java/uk/gov/hmcts/reform/prl/controllers/barrister/BarristerController.java

```diff
@@ -77,16 +81,21 @@ public AboutToStartOrSubmitCallbackResponse handleAddAboutToStartEvent(
                 .errors(errorList)
                 .data(caseDataUpdated)
                 .build();
-        } else {
-            throw (new InvalidClientException(INVALID_CLIENT));
+        } catch (Exception e) {
+            log.error("Error in handleAddAboutToStartEvent", e);
+            return AboutToStartOrSubmitCallbackResponse.builder()
+                .errors(List.of("An unexpected error occurred: " + e.getMessage()))
```

Comment:
Just checking - is this going to be a message we want to be display to the user?

Reply:
I'll try reuse something more appropriate